plugins/extrusions/README.md

```diff
@@ -0,0 +1,20 @@
+# Extrusions
+
+Aluminium extrusions in a variety of sizes in both standard and v-slot profiles.
+Currently e2020, e2040, e2080, e4040, v2020, v2040, and v4040 extrusions are available.
```

Comment:
V stands for V-slot I assume. What would e stand for? Is it some "standardized" extrusion type. For sure there are more, but I know of Bosch-type and Item-type extrusions.

Reply:
E just stands for extrusion, meaning a generic type. It's the same abbreviation I used when I implemented extrusions in NopSCADlib, see https://github.com/nophead/NopSCADlib#Extrusions .

Someone might implement the B and I types in future, if the exact profile was important for their project, but for most projects a generic form is perfectly acceptable (and I have found variations in the profile form different suppliers, especially the size of the center hole and the channel width).

Reply:
Maybe good to state somewhere what is the source of this "generic" extrusion. I assume you did not just imagine the dimensions yourself.

Reply:
OK, I'll state the original source.